scss/_button-group.scss

```
@@ -41,7 +41,7 @@
   }
 
   // Reset rounded corners
-  > .btn:not(:last-child):not(.dropdown-toggle),
+  > .btn:not(:last-child):not(.dropdown-toggle, .dropdown),
```

We cannot use the list argument syntax here since [it's not fully supported](https://caniuse.com/css-not-sel-list) by [our current browsers target](https://github.com/twbs/bootstrap/blob/main/.browserslistrc).

```suggestion
  > .btn:not(:last-child):not(.dropdown-toggle):not(.dropdown),
```